### DISCUSSION TITLE
Variant of System Status Icon

### DISCUSSION THREAD

**Discussion by tonge495** [twilio-labs/paste#3724] [Request: Icon] [answered]
### Description

Hi,

Our team is planning to use a variation of the system status icon for when there is a new notification that requires the user's attention.  Is there a way to get this incorporated as part of the Paste icon?  This will be the same as Flex's system status icon variant:

![image](https://github.com/twilio-labs/paste/assets/9394413/75939ee3-fdaf-469a-8c1b-4e42e8b85a1e)

### Link to icon

https://paste.twilio.design/components/icon

### Icon type

- [X] UI icon
- [ ] Product icon

**Comment by serifluous** [comment, marked as answer]
Hey @tonge495 ! Can you submit a dependency request for this for the "corner ornament component"? https://airtable.com/appwvsQKyJHfnG5Sa/shrvMvL3BG6Zm0f9l

The way you'll implement this is using the SystemStatusIcon and then a corner ornament component (on roadmap) on top of it.

**Reply by tonge495** [reply]
Will do thank you!
